src/Module/Contact.php

```diff
@@ -947,7 +947,7 @@ private static function getConversationsHMTL($a, $contact_id, $update)
 		}
 
 		if (DBA::isResult($contact)) {
-			$a->page['aside'] = '';
+			DI::page()['aside'] = '';
```

Comment:
Is this working and valid?

Reply:
Yes, the `Page` class implements the `ArrayAccess` interface which allows array-style access to class members.

Reply:
I really would prefer dedicated methods for that :-)

Reply:
Yep, me too, but it can come in later.